### PR TITLE
chore(env checker): add bicep cli support - 1

### PR DESCRIPTION
**Pull request by meifans**
add depsChecker files to solution

### CODE REVIEW

**Review by a1exwang — COMMENTED** [review]
what is the empty file named `placeholder` for?

File: packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    }
+    return "bicep";
+  }
+
+  private getBicepBitSuffixName(): string {
+    if (isWindows()) {
+      return "bicep-win-x64.exe";
+    }
+    if (isMacOS()) {
+      return "bicep-osx-x64";
```
Comment (a1exwang) [inline packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts:270]: Does bicep support M1? If not, maybe detect M1 Mac and explicitly tell it to the user

Comment (meifans) [reply packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts:270]: > what is the empty file named `placeholder` for?

removed

Comment (meifans) [reply packages/fx-core/src/plugins/solution/fx-solution/utils/depsChecker/bicepChecker.ts:270]: > Does bicep support M1? If not, maybe detect M1 Mac and explicitly tell it to the user

`bicep` work well on my mac m1, but it available for all mac m1 is to verify.
If bicep not support mac m1, the version of ARM need to have a workaround for m1 user.

**Review by a1exwang — APPROVED** [review]

**Pull request merged**